src/OOVPADatabase/D3D8LTCG/3925.inl

```diff
@@ -0,0 +1,44 @@
+// ******************************************************************
+// *
+// *   OOVPADatabase->D3D8LTCG->3911.inl
+// *
+// *  XbSymbolDatabase is free software; you can redistribute them
+// *  and/or modify them under the terms of the GNU General Public
+// *  License as published by the Free Software Foundation; either
+// *  version 2 of the license, or (at your option) any later version.
+// *
+// *  This program is distributed in the hope that it will be useful,
+// *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+// *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// *  GNU General Public License for more details.
+// *
+// *  You should have recieved a copy of the GNU General Public License
+// *  along with this program; see the file COPYING.
+// *  If not, write to the Free Software Foundation, Inc.,
+// *  59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+// *
+// *  (c) 2017 jarupxx
+// *
+// *  All rights reserved
+// *
+// ******************************************************************
+
+// test case: NASCAR Heat 2002
```

Comment:
nit: Move below the title symbol comment, not above. testcase is one word for internal documentation.